### ISSUE TITLE
[Accessibility: ACC-270.1] List item: UI cannot be accessed.

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12562]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
List items within timeline (both horizontal and vertical) are not accessible with keyboard.
Each & every UI, must be accessible using the keyboard.
Keyboard is the prerequisite for support of assistive technologies to enable users with disabilities to read and operate software applications, hence all the UIs must be accessible with keyboard.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest versions

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [Core Timeline](https://sap.github.io/fundamental-ngx/#/core/timeline)
Use timeline component and try to access using keyboard, no list item can be accessed.

#### Please provide relevant source code (if applicable).
You can refer ngx example, it is reproducible.

#### Please provide stackblitz example(s).
You can refer ngx example, it is reproducible. (https://stackblitz.com/run?file=src%2Fapp%2Ftimeline-basic-example.component.ts)

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@divyamalhotra488 the stackblitz example is incorrect.

**Comment by droshev** [comment]
closing as there was no activity
